### ISSUE TITLE
Typing versus pasting in an input coordinate produces invalid results (ArcMap)

### ISSUE THREAD

**Issue by elinz** [Esri/coordinate-conversion-addin-dotnet#248]
- Type in the following coordinate: 90N 50E
The coordinate is interpreted and the following output is created:
![image](https://cloud.githubusercontent.com/assets/4635490/21835339/173b656c-d771-11e6-9c27-320257d50935.png)
The output coordinates DD, DDM, DMS, and GARS are correct, however MGRS (32NNF0000000000), USNG (32NNF0000000000), and UTM (32N 500000 0) are not correct.

The output MGRS/USNG/UTM values represent 0.0S 9.0E (also confirmed by Geotrans).

- Now remove the coordinate from the input textbox and _paste_ in the same value that was typed in above: 90N 50E
Pasting in the value does not create the invalid MGRS, USNG, and UTM values:
![image](https://cloud.githubusercontent.com/assets/4635490/21835522/81eaa44e-d772-11e6-8f7f-74fb4eb334b0.png)

SIDENOTE: UTM is only valid between 84N and 80S. Outside of these latitudes, the Universal Polar Stereographic (UPS) coordinate system is used (http://earth-info.nga.mil/GandG/coordsys/grids/utm.html)

**Comment by lfunkhouser** [comment]
War room update: Team agreed that coordinate will not be converted until pressing Enter.

**Comment by pHill5136** [comment]
@jmhauck Noticed when you paste bogus coordinate (90N 50E) and hit enter sometimes map flashes at 0 0 other times crashes arcmap when the flash is created. Thought you should be aware since you are creating an onEnter event...

**Comment by dfoll** [comment]
@Dbarnes1  this means we need to check to make sure we instruct the user to press enter

**Comment by elinz** [comment]
@nfeuerstein @Dbarnes1 @dfoll Verified that input coordinates are not evaluated until the user clicks enter. I also verified that the user doc instructs the user to click Enter for the coordinate to be converted. See below:
![image](https://cloud.githubusercontent.com/assets/4635490/21950923/7dba0272-d9b4-11e6-8cd0-8423931e9709.png)

@nfeuerstein As far as I everything to address this issue was been completed. I've verified that when users enter a coordinate into the Input box that the coordinate only gets converted when the user clicks Enter and confirmed that we instruct the user to click Enter. Was there something else that needed to be done? Just trying to remember why the milestone was changed to Version - After Next. Thanks.
